packages/k4rectracker/package.py

```diff
@@ -32,6 +32,7 @@ class K4rectracker(CMakePackage, Key4hepPackage):
     depends_on("k4fwcore")
     depends_on("marlinutil")
     depends_on("root")
+    depends_on("delphes") # To be updated with specifc version once new delphes tag with latest changes is released
```

Comment:
```suggestion
    depends_on("delphes", when="@0.6.0:") # To be updated with specifc version once new delphes tag with latest changes is released
```
Let's assume the next version of k4RecTracker is 0.6.0, we don't need to build delphes in case we were going to build previous versions.